packages/commercetools/api-client/src/api/isLoggedIn/index.ts

```diff
@@ -0,0 +1,13 @@
+import { isUserSession } from '../../helpers/utils';
+
+const isLoggedIn = (context) => {
+  const { config } = context;
+
+  if (config.handleIsLoggedIn) {
```

Comment:
It's better to check if `handleIsLoggedIn` type is a function

Reply:
Updated